knit/core.py

```diff
@@ -423,7 +444,7 @@ def logs(self, shell=False):
     def print_logs(self, shell=False):
         """print out a more console-friendly version of logs()"""
         for l, v in self.logs(shell).items():
-            print('Container ', l, ', id ', v.get('id', 'None'), '\n')
+            print('\n### Container ', l, ', id ', v.get('id', 'None'), ' ###\n')
```

Comment:
Why this change?

Reply:
Makes the printout easier to see, purely visual.